packages/app/src/specs/banners/TrackedBanner.cy.tsx

```diff
@@ -25,7 +25,7 @@ describe('<TrackedBanner />', () => {
 
     // Initially mount as visible
     // @ts-ignore
-    cy.mount({ render: () => <TrackedBanner data-cy="banner" bannerId="test-banner" v-model={shown.value} hasBannerBeenShown={false} /> })
+    cy.mount({ render: () => <TrackedBanner data-cy="banner" bannerId="test-banner" v-model={shown.value} hasBannerBeenShown={false} eventData={{} as any}/> })
```

Comment:
Should we just pass in some real data? The interface is pretty simple:

```js
type EventData = {
  campaign: string
  medium: string
  cohort?: string
}
```

Reply:
Just following existing pattern where the values of this eventData do not matter for this particular test.